app/client/jsx/Room.jsx

```diff
@@ -179,7 +197,27 @@ class Room extends Component {
         this.socketApi.leaveRoom(this.props.user.userId, this.state.room)
     }
 
+    computeMapState(room) {
+        const mapAlreadyUnlocked = LocalStorage.get("MAP_UNLOCKED")
+        const mapUnlockThreshold = 3
+        const visitedRooms = Object.values(this.props.visited).filter(x => x).length
+        const isMapUnlocked = mapAlreadyUnlocked ||
+            (_.has(this.roomTypesWithMap, room.type) && (visitedRooms >= mapUnlockThreshold))
+        const showMapTooltip = !mapAlreadyUnlocked && isMapUnlocked
+        LocalStorage.set("MAP_UNLOCKED", isMapUnlocked)
```

Comment:
I think it might be better to prefix all of our `localStorage` keys with `JITSI_APP` or something app-specific, or wrap them all in an app-specific object. It seems like right now if I print out `localStorage` I'll see `USER` and `MAP_UNLOCKED` and whatever else we're throwing in there which feels a little messy